sdcm/mgmt/cli.py

```diff
@@ -1302,6 +1323,17 @@ def client_version(self):
     def parsed_client_version(self):
         return LooseVersion(self.client_version)
 
+    @property
+    def client_version_timestamp(self) -> int:
+        """Gets the timestamp of the client version
+
+        Example, for client version `3.3.3-0.20240912.924034e0d` the timestamp is `1726099200` (2024-09-12 00:00:00)
+        """
+        date_str = self.client_version.split('.')[-2]
+        date_obj = datetime.datetime.strptime(date_str, "%Y%m%d")
+        timestamp = int(time.mktime(date_obj.timetuple()))
```

Comment:
same can be done with `date_obj.timestamp()`

Reply:
Fixed